src/app/analyzer.service.ts

```diff
@@ -109,6 +111,19 @@ export class AnalyzerService {
     return detail.title + ' (' + detail.geography.shortName + '; ' + detail.frequencyShort + ')';
   }
 
+  formatChartName(detail) {
+    if (detail.seasonalAdjustment === 'seasonally_adjusted') {
+      return detail.title + ' (' + detail.geography.shortName + '; ' + detail.frequency + '; Seasonally Adjusted)';
+    }
+    if (detail.seasonalAdjustment === 'not_seasonally_adjusted') {
+      return detail.title + ' (' + detail.geography.shortName + '; ' + detail.frequency + '; Not Seasonally Adjusted)';
+    }
+    if (detail.seasonalAdjustment === 'not_applicable') {
+      return detail.title + ' (' + detail.geography.shortName + '; ' + detail.frequency + ')';
+    }
+    return detail.title + ' (' + detail.geography.shortName + '; ' + detail.frequency + ')';
+  }
```

Comment:
Let's make this a bit more DRY:
```
formatChartName({ title, geography, frequency, seasonalAdjustment }) {
  let ending = '';
  if (seasonalAdjustment == 'seasonally_adjusted') {
    ending = '; Seasonally Adjusted';
  }
  if (seasonalAdjustment == 'not_seasonally_adjusted') {
    ending = '; Not Seasonally Adjusted';
  }
  return `${title} (${geography.shortName}; ${frequency}${ending})`
}
```
uses: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment